<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugConsoleDialog</class>
 <widget class="QDialog" name="DebugConsoleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug console</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="debugEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Command:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="commandEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="queueLabel">
       <property name="text">
        <string>Queue: 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
